src/OpenLoco/src/Objects/Object.h

```diff
@@ -109,6 +109,11 @@ namespace OpenLoco
         {
             return !(*this == rhs);
         }
+
```

Comment:
I don't think this needs to exist here. This is only relevant to that window.

Reply:
I initially tried defining it locally, but my compiler complained that it wasn't part of the actual struct. Short of extending the ObjectHeader type (e.g. into SortableObjectHeader), any suggestions?